packages/ui/src/components/va-tree-view/VaTreeView.demo.vue

```diff
@@ -1,59 +1,252 @@
 <template>
   <VbDemo>
-    <VbCard>
-      <va-button @click="$refs.treeView.expand()">
-        EXPAND ALL
-      </va-button>
-      <va-button @click="$refs.treeView.collapse()">
-        COLLAPSE ALL
-      </va-button>
-
-      <va-tree-view ref="treeView">
-        <va-tree-category
-          label="Category text"
-          is-open
-        >
-          <va-tree-node>
-            Node text
-          </va-tree-node>
-          <va-tree-category
-            label="Category text"
-            is-open
-          >
-            <va-tree-node>
-              Node text
-            </va-tree-node>
-          </va-tree-category>
-        </va-tree-category>
+    <VbCard title="Default">
```

Comment:
<img width="206" alt="image" src="https://user-images.githubusercontent.com/5394573/187558359-0b9fd975-708b-4782-9b22-bb94ac83bdfe.png">
  
Does it make sense that childe nodes have left padding without icon? Can we check other frameworks?

Reply:
I've kept it like in the figma. If it's not necessary, we can move it to new issue? As a little fix I can try to reduce the width of the highlighting background.
![image](https://user-images.githubusercontent.com/29167241/187635462-0d39c3f4-bf37-441f-933c-55be9e2d43f7.png)

Reply:
There should be orange selection on mouse click (only on keyboard navigation).

![image](https://user-images.githubusercontent.com/5394573/191754946-654095d1-0ec2-447f-b1ee-a7bc2a0b5bab.png)